README.md

```diff
@@ -94,7 +94,7 @@ It supports the Uniris Cryptography rules which are:
 
   ```js
   const uniris = require("uniris")
-  const publicKey = uniris.derivePublicKey("mysuperpassphraseorseed", 0)
+  const publicKey = uniris.deriveKeyPair("mysuperpassphraseorseed", 0)
```

Comment:
I think the method `derivePublicKey`  is obsolete, this part of the doc should be removed.